src/chrome/content/content.js

```diff
@@ -14,16 +14,7 @@ function EscapeHTML(string) {
 }
 
 function MediaDetails(media) {
-  var text = "";
-  var found = false;
-
-  if (media.length > 0) {
-    if (found) {
-      text += "\n";
-    }
-    found = true;
-    text += EscapeHTML(content.document.location) + "\n";
-  }
+  var text = EscapeHTML(content.document.location) + "\n";
```

Comment:
MediaDetails() is called only when media.length > 0. So this will never be the last line with tailing CR.